cms/static/sass/elements-v2/_header.scss

```diff
@@ -132,7 +137,6 @@
 
     .nav-account-user {
       .title {
-        max-width: ($baseline*10.5);
```

Comment:
Seems odd that this just got deleted?

Reply:
I deleted it because it's redeclared on 137